src/lib/network/network_autoyast.rb

```diff
@@ -120,7 +121,36 @@ def set_network_service
       NetworkService.EnableDisableNow
     end
 
-    private
+    # Initializates DNS setup according AY profile
+    #
+    # FIXME: it currently don't write DNS configuration. It is used for initialization
+    # of DNS setup according AY profile in 1st stage as part of network setup was moved
+    # here already and some parts of network configuration needs to know it. DNS write
+    # is still done in 2nd stage.
```

Comment:
If a public method returns something, we should document what the return value means.